bin/cron/upload_to_firebase

```diff
@@ -44,10 +46,31 @@ def get_weather_data
   return records
 end
 
+def parse_csv(filename)
+  records = []
+  id = 1
+  table = CSV.parse(File.read(filename), headers: true)
+  table.each do |row|
+    record = OpenStruct.new
+    record.id = id
+    id += 1
+    table.headers.each do |col|
+      record[col] = row[col]
+    end
+    records.push(record.to_h.to_json)
+  end
+  return records
+end
+
 def main
   fb = FirebaseHelper.new('shared')
-  weather_records = get_weather_data
-  fb.upload_shared_table('openweathermap', weather_records)
+  if ARGV.empty?
+    records = get_weather_data
+    fb.upload_shared_table('openweathermap', records)
```

Comment:
Why is weather a special case?

Reply:
It's not exactly a special case, it's just the first API we have.
I'm imagining the format of this will change as we add more data sources.
For now, though, there are only two types: CSVs (so you would need to supply the filename) and weather.

Reply:
👍